### DISCUSSION TITLE
🔨iLogtail开发者双周会（1月4日 19:00-20:00）

### DISCUSSION THREAD

**Discussion by linrunqi08** [alibaba/ilogtail#585] [Announcements]
时间：1月4日 周三 19:00 - 20:00
会议号：580 339 70878
电话呼入：
+865965013949（中国大陆）
+862281944293（中国大陆）
入会链接：https://meeting.dingtalk.com/j/UHR9VcjbgUh

议题：
1. pulsar flusher进展同步
2.  influxdb proxy场景分享
3. Java Continuous Profiling 调研与设计分享

**Comment by shalousun** [comment]
flusher pulsar支持设计讨论 , https://github.com/alibaba/ilogtail/discussions/598
